app/src/main/java/ar/com/wolox/android/features/homepage/news/details/NewsDetailsActivity.kt

```diff
@@ -0,0 +1,58 @@
+package ar.com.wolox.android.features.homepage.news.details
+
+import android.content.Context
+import android.content.Intent
+import ar.com.wolox.android.R
+import ar.com.wolox.android.databinding.NewsDetailsActivityBinding
+import ar.com.wolox.android.features.homepage.news.deltaTime
+import ar.com.wolox.android.models.NewFromPage
+import ar.com.wolox.wolmo.core.activity.WolmoActivity
+
+class NewsDetailsActivity() : WolmoActivity<NewsDetailsActivityBinding>() {
+
+    override fun layout() = R.layout.news_details_activity
+
+    override fun init() {
+        replaceFragment(binding.activityBaseContent.id, NewsDetailsFragment.newInstance(
+            intent.getIntExtra(ID, 0),
+            intent.getStringExtra(COMMENTER),
+            intent.getStringExtra(COMMENT),
+            deltaTime(intent.getStringExtra(CREATED_AT)),
+            intent.getBooleanExtra(LIKES, false)
```

Comment:
You can pass the entire New model as a Serializable here instead of passing every field separate